.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "wadm",
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
+	"features": {
+		"ghcr.io/devcontainers/features/rust:1": {
```

Comment:
If this devcontainer is meant to work in, you may want to include a few more features like: 
- <https://github.com/devcontainers/features/tree/main/src/common-utils>